test/server/server_corpus/clusterfuzz-testcase-server_fuzz_test-4832853025095680

```diff
@@ -25,7 +25,7 @@ static_resources {
     hosts {
       socket_address {
         address: "127.0.0.1"
-        port_value: 4294312379
```

Comment:
Doesn't this defeat the purpose of this test?

Reply:
Probably. I'll relax this one.